judo-runtime-core-guice/src/main/java/hu/blackbelt/judo/runtime/core/guice/JudoModelLoader.java

```diff
@@ -107,14 +154,14 @@ public static JudoModelLoader loadFromDirectory(String modelName, File directory
             throw new IllegalArgumentException("Given file is not directory: " + directory);
         }
 
-        return loadFromURL(modelName, directory.toURI(), dialect, validate);
+        return loadFromURL(modelName, directory.toURI(), dialect, validate, loaKeycloak);
     }
 
-    public static JudoModelLoader loadFromURL(String modelName, URI uri, Dialect dialect) throws Exception {
-        return loadFromURL(modelName, uri, dialect, true);
+    public static JudoModelLoader loadFromURL(String modelName, URI uri, Dialect dialect, boolean loaKeucloak) throws Exception {
+        return loadFromURL(modelName, uri, dialect, true, loaKeucloak);
     }
 
-    public static JudoModelLoader loadFromURL(String modelName, URI uri, Dialect dialect, boolean validate) throws Exception {
+    public static JudoModelLoader loadFromURL(String modelName, URI uri, Dialect dialect, boolean validate, boolean loadKetkloak) throws Exception {
```

Comment:
loaKeycloak, loaKeucloak, loadKetkloak -> I guess this should be loadKeycloak :D